Add-ons/UmbracoCourier/Developer/ResourceResolvers.md

```diff
@@ -0,0 +1,12 @@
+# ResourceResolvers
+A resourceresolver is slightly different from dataresolvers, as they only triggers when a resource/file is packaged and extracted. This means it can only modify the file itself, but not call back to the item, which the resource belongs to.
```

Comment:
```suggestion
A resourceresolver is slightly different from dataresolvers, as they only trigger when a resource/file is packaged and extracted. This means it can only modify the file itself, but not call back to the item, which the resource belongs to.
```